docs/contributing/feature-flags.md

```diff
@@ -254,10 +254,54 @@ the retrieval methods:
 
 ## Feature flag lifecycle
 
+:::tip LaunchDarkly access
+
 Let your management know when you need to change something about a feature online inside
 LaunchDarkly. Only a small number of users have accounts with LaunchDarkly to save on licensing
 costs.
 
+:::
+
+The process for updating flag values and the ownership of flag changes depends on the environment
+and change as a flag proceeds through our cloud landscape.
```

Comment:
⛏️ "cloud landscape" sounds beautiful but perhaps too whimsical for docs 😜

Reply:
Updated in [`151b341` (#389)](https://github.com/bitwarden/contributing-docs/pull/389/commits/151b341f9f5d78b41d6a78d3315ed79f0659e45d) to be a bit less poetic.